packages/http/libraries/core/src/http/adapter/InversifyHttpAdapter.ts

```diff
@@ -133,7 +133,8 @@ export abstract class InversifyHttpAdapter<
         ]?.(...handlerParams);
 
         return this.#reply(req, res, value, statusCode);
-      } catch (_error: unknown) {
+      } catch (error: unknown) {
+        this.#logger.error((error as Error).stack ?? (error as Error).message);
```

Comment:
`error` might not be an `Error`. JS allow throwing any value like `throw 3`. A typeguard should be added to detect `Error` errors to apply the current log. Otherwise, a generic error might be logged instead.